sormas-backend/src/main/java/de/symeda/sormas/backend/person/PersonFacadeEjb.java

```diff
@@ -392,8 +399,82 @@ public Date getLatestFollowUpEndDateByUuid(String uuid) {
 		cq.orderBy(cb.desc(contactRoot.get(Contact.FOLLOW_UP_UNTIL)));
 
 		List<PersonFollowUpEndDto> resultlist = em.createQuery(cq).getResultList().stream().distinct().collect(Collectors.toList());
-		return resultlist.get(0).getLatestFollowUpEndDate();
```

Comment:
This method needs to also consider FollowUpEndDates of cases here, if i'm not mistaken...

Reply:
So should the latest follow up end date from both contacts and cases related to the person be returned?
What about the method above, `getLatestFollowUpEndDates`? Should we return a list with dates combined from both cases and contact?

Reply:
Yes, good point. Each of those methods should return the latest follow up end date for persons, no matter if they come from a contact or a case.